prime-router/settings/STLTs/WA/wa-phd.yml

```diff
@@ -79,11 +79,13 @@
               - "Bundle.entry.resource.ofType(Patient).where(address.city.empty()).count() = 0"
               - "Bundle.entry.resource.ofType(Patient).where(address.state.empty()).count() = 0"
               # "isValidCLIA(testing_lab_clia,reporting_facility_clia)"
-              - "Bundle.entry.resource.ofType(ServiceRequest).performer.resolve().identifier.where(type.coding.code = 'CLIA').value.getIdType() = 'CLIA'"
-              - "Bundle.entry.resource.ofType(ServiceRequest).requester.resolve().organization.resolve().identifier.where(type.coding.code = 'CLIA').value.getIdType() = 'CLIA'"
+              - "(Bundle.entry.resource.ofType(ServiceRequest).performer.resolve().identifier.where(type.coding.code = 'CLIA').value.getIdType() = 'CLIA') or (Bundle.entry.resource.ofType(Organization).identifier[1].where(type.coding.code = 'CLIA').value.getIdType() = 'CLIA')"
```

Comment:
I found a different way to write this quality filter that works for both Carbon Health and Simple Report and simplifies it
  `"Bundle.entry.resource.ofType(Organization).identifier.where(type.coding.code = 'CLIA' or system = 'CLIA').where(value.getIdType() != 'CLIA').count() = 0",`